lib/services/ios-project-service.ts

```diff
@@ -657,6 +657,10 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		return null;
 	}
 
+	public cleanProject(projectRoot: string, options: string[]): IFuture<void> {
+		return null;
```

Comment:
It's better to use:
```TypeScript
return Future.fromResult();
```

This way the code will not fail in case the method is called (currently in case you call `cleanProject(..).wait()` it will fail that it cannot read property wait fo null.